File: Math/codeforces_231A.py

# Problem link ===>> https://codeforces.com/problemset/problem/231/A

# Read number of problems
n = int(input())

# Count the number of implemented problems
problems = 0

# Iterate for each problem
for i in range(n):
    # Read if the input data for each problem
    friends = input().split()
    # If two of more of the friends are sure about the problem implement it
    if friends.count("1") >= 2:
        problems += 1
        
print(problems)

File: Math/codeforces_231A.cpp

// Problem link ===>> https://codeforces.com/problemset/problem/231/A
// submission link ===>> https://codeforces.com/contest/231/submission/206754658

#include <iostream>
using namespace std;

int main()
{
    int n;
    cin >> n;

    int count = 0;
    for (int i = 0; i < n; i++)
    {
        int p, v, t;
        cin >> p >> v >> t;

        // Count the number of problems the team can solve together
        if (p + v + t >= 2)
        {
            count++;
        }
    }

    cout << count << endl;

    return 0;
}
